CRM/Core/Payment/BaseIPN.php

```diff
@@ -137,31 +137,6 @@ public function validateData($input, &$ids, &$objects, $required = TRUE, $paymen
     return TRUE;
   }
 
-  /**
-   * Load objects related to contribution.
-   *
-   * @deprecated
-   *
-   * @input array information from Payment processor
-   *
-   * @param array $input
-   * @param array $ids
-   * @param array $objects
-   * @param bool $required
-   * @param int $paymentProcessorID
-   *
-   * @return bool|array
-   * @throws \CRM_Core_Exception
-   */
-  public function loadObjects($input, &$ids, &$objects, $required, $paymentProcessorID) {
```

Comment:
@eileenmcnaughton you also need to drop the loadObject call in L134 above!

Reply:
@monishdeb thanks - done